### ISSUE TITLE
Suggestions and possible typos

### ISSUE THREAD

**Issue by learnbyexample** [Textualize/textual#1159]
1) Suggestion - use light theme for CSS code snippet when switching theme for the `python -m textual` demo:

![image](https://user-images.githubusercontent.com/17766317/201273839-cb1cceae-8f22-4ed1-9587-8ff30c836398.png)

<br><br>

2) Terminal title in screenshots for https://textual.textualize.io/tutorial/ - would suggest screenshots discussed under `stopwatch01.py` to use the script name at the top

![image](https://user-images.githubusercontent.com/17766317/201273054-56e3faa0-7a7c-49f4-a550-e1b194896e02.png)
![image](https://user-images.githubusercontent.com/17766317/201273090-baf1a3ec-eb7e-49a8-a1fd-10b033cd9a16.png)

02 to 06 have script name at the top, for example:

![image](https://user-images.githubusercontent.com/17766317/201273180-3de63beb-08f0-4c4f-95fa-c8c9cda8b112.png)

<br><br>

3) https://textual.textualize.io/tutorial/#css-basics illustration: `background color is $panel-darken-1` in the figure but the CSS code uses `background: $boost;`

![image](https://user-images.githubusercontent.com/17766317/201274202-fec825dc-7c94-4863-8b4f-ecde269aa439.png)

<br><br>

4) https://textual.textualize.io/guide/actions/#screenshot - description column says `~/` and default column says `./`

![image](https://user-images.githubusercontent.com/17766317/201274672-effc9f6e-37ad-4cd0-af7e-8935cc705a94.png)

**Comment by davep** [comment]
Good catch on `4`, fixed that while tidying some other docstring things: bc851d5466679723e4725194fe7987648f65b753

(others will likely get reviewed early next week as they're in the docs themselves, but `4` was a docstring and I happened to be passing that way)

**Comment by willmcgugan** [comment]
Fixed, with the exception of the color theme changing code. Which we might do at some point in the future.
